src/doom/g_game.c

```diff
@@ -1197,7 +1226,8 @@ void G_Ticker (void)
 
 	    memcpy(cmd, &netcmds[i], sizeof(ticcmd_t));
 
-	    if (demoplayback) 
+        // [crispy] don't read input from demo when paused
+	    if (demoplayback && !paused)
```

Comment:
Will pausing also work in demo continue mode?

Reply:
There's no reason to think it wouldn't, but the thing is, how do I test it? Is it `-playdemo demoname1 -record demoname2`, using the pause key during the blank screen stage and then taking over with key_demo_quit? 

I tried just that with a demo that went from E1M1 to E1M4 of a PWAD and took over at the intermission after E1M4. Went into the E1M5 of the IWAD with a rocket launcher and a plasma rifle from the previous levels, played for a minute and quit with key_demo_quit. The end of the resulting demo (the E1M5 part) plays back fine with prb+ using -skipsec.

I tried the same with `-warp 1 2` so that I could take over at an arbitrary point toward the middle of the demo, and it seemed to work - I was able to take over - but no demoname2 file was created.